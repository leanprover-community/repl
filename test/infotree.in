{"cmd": "def f (h : Int) : Nat := by constructor", "infotree": "substantive"}

