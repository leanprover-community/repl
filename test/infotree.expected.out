{"messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 7},
   "endPos": {"line": 1, "column": 8},
   "data": "unused variable `h` [linter.unusedVariables]"}],
 "infotree":
 [{"node":
   {"stx":
    {"range":
     {"synthetic": false,
      "start": {"line": 1, "column": 28},
      "finish": {"line": 1, "column": 39}},
     "pp": "constructor"},
    "name": "Lean.Parser.Tactic.constructor",
    "goalsBefore": ["h : Int\n‚ä¢ Nat"],
    "goalsAfter": []},
   "kind": "TacticInfo",
   "children": []}],
 "env": 0}

